CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Add remote routing table for remote state publication with experimental feature flag ([#13304](https://github.com/opensearch-project/OpenSearch/pull/13304))
 - [Remote Store] Add support to disable flush based on translog reader count ([#14027](https://github.com/opensearch-project/OpenSearch/pull/14027))
 - [Query Insights] Add exporter support for top n queries ([#12982](https://github.com/opensearch-project/OpenSearch/pull/12982))
+- [Query Insights] Support customized labeling for search queries ([#13374](https://github.com/opensearch-project/OpenSearch/pull/13374))
```

Comment:
I would adjust this description (and the PR title) to make it clear that we're **only** adding `X-Opaque-Id` to the search request metadata for query insights.

Reply:
Let me update the change log.